src/posthog-featureflags.ts

```diff
@@ -356,39 +368,63 @@ export class PostHogFeatureFlags {
         this._fireFeatureFlagsCallbacks(errorsLoading)
     }
 
-    /*
-     * Override feature flags on the client-side.  Useful for setting non-persistent feature flags, or for testing/debugging
-     * feature flags in the PostHog app.
+    /**
+     * @deprecated Use overrideFeatureFlags instead. This will be removed in a future version.
```

Comment:
added the deprecated tag, so users of this latest version will see the warning in their IDE like this
<img width="1330" alt="image" src="https://github.com/user-attachments/assets/2cda80d2-ac8c-45a4-a090-f8b421bde022" />